src/frontend/src/styles/main.css

```diff
@@ -2074,6 +2074,16 @@ input[type="checkbox"] {
   user-select: none;
 }
 
+/* Wrap QR codes in a white square (for legibility) and prettify a bit */
+.c-qrcode canvas {
+  background: white;
+  padding: calc(2 * var(--vs-gutter));
+  border: 2px solid var(--vc-brand-purple);
+  border-radius: 2px;
+  /* Ensure the canvas doesn't overflow on small screens */
+  max-width: 100%;
```

Comment:
@meodai I had an issue on small screen, because the canvas sets its own size (to 256 pixels). This seemed like a good fix but maybe there's a better way?

Reply:
Yes, good idea!